docs/Reference/architecture/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING
+
+## Installation
+
+Cosmos Cash diagrams are based on PlantUML. 
+
+PlantUML can be installed on your local system through your platforms' package manager:
+
+MacOS, through [Homebrew](https://brew.sh/)
+
+```
+> brew install plantuml
+```
+
+Windows, through [Chocolatey](https://chocolatey.org/)
+
+```
+> choco install plantuml
+```
+
+Linux-based systems, through your distributions package manager
+
+## Contributing diagrams
+
+The source code for all diagrams SHOULD be stored in `./src` folder AND have the `.puml` suffix. Please refer to the [PlanUML documentation](https://plantuml.com) for further details.
+
+## Themes
+
+Where possible all diagrams will be rendered to have a similar look and feel. To achieve this add the following to your diagram:
+
+```
+@startuml myDiagram
+!theme tendermint from ../themes/
+...
+
+@enduml
+```
+
+## How to generate diagram images
+
+There are [many ways to run PlantUML](https://plantuml.com/running) to generate images from the `.puml` files, but this project prefers the following: 
+
+* PlantUML CLI
+* Embed PlantUML diagrams using PlantUML's proxy service in Markdown
+* Visual Studio Code Plugin - best for development
+
+### PlantUML CLI
+
+There is [makefile](.makefile) that will build all `src/*.puml` files. To create SVG images just run `make (svg|png)` as needed. `make clean` will get a clean folder ready for new build.
+
+This is the preferred and cleanest way to create all the architecture documentation.
+
+### Proxy service
+
+This method uses PlantUML's [proxy service](https://plantuml.com/server) to put generate images and embed into a Markdown page. 
+
+To use the proxy service integration simply use:
```

Comment:
```suggestion
To use the proxy service integration, use:
```

Reply:
it's tempting when we think something is easy, but let's omit "simply" in all cases, per https://developers.google.com/style/tone?hl=en#some-things-to-avoid-where-possible